core/src/stored/backends/gfapi_device.cc

```diff
@@ -338,20 +342,23 @@ int gfapi_device::d_open(const char* pathname, int flags, int mode)
           switch (device_options[i].type) {
             case argument_uri:
               gfapi_uri_ = bp + device_options[i].compare_size;
-              done = true;
+              break;
+            case argument_logfile:
+              gfapi_logfile_ = bp + device_options[i].compare_size;
+              break;
+            case argument_loglevel:
+              gfapi_loglevel_ =
```

Comment:
In case of error  gfapi_loglevel_ will be zero.

Reply:
this is expected and not a problem.